src/homework10/FileNavigator.java

```diff
@@ -0,0 +1,97 @@
+package homework10;
+import java.util.*;
+//0. Створити клас FileNavigator. Клас призначений для зберігання списку файлів, які розташовані на конкретному шляху.
+//Наприклад: /path/to/file-> [files.txt, firstApp.java]
+
+public class FileNavigator {
+
+    public final Map<String, ArrayList<FileData>> files = new HashMap<>();
```

Comment:
Why do you use `ArrayList` - are you using some special functionality of `ArrayList`? if not - make it more abstract - `List`